etc/profile-m-z/makedeb.profile

```diff
@@ -0,0 +1,24 @@
+# Firejail profile for makedeb
+# Description: A simplicity-focused packaging tool for Debian archives
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include makedeb.local
+# Persistent global definitions
+#include globals.local
+
+# Note: see this Arch forum discussion https://bbs.archlinux.org/viewtopic.php?pid=1743138
+# about makepkg for potential issues and their solutions when Firejailing makedeb, 
+
+# This profile could be significantly strengthened by adding the following to makedeb.local
+# whitelist ${HOME}/<Your Build Folder>
+# whitelist ${HOME}/.gnupg
+
+# noroot is only disabled to allow the creation of kernel headers from an official PKGBUILD.
+#noroot
+
+ignore include makepkg.local
```

Comment:
```suggestion
```

Since this profile redirects to makepkg.profile, leave these lines only in the
latter to avoid duplication and make maintenance easier.

Reply:
@pirate486743186 [on Aug 9](https://github.com/netblue30/firejail/pull/5302#discussion_r941111298):

> The comments are meant to be read from users. So i think they should stay
> there.

If users read the comments on redirect profiles, they should eventually come
across the `# Redirect` comment, in which case they can follow the include to
see what else the profile does.

Also, avoiding such duplication is how it's done in other redirect profiles.

See for example the commented whitelisting of the `${HOME}/Projects` directory,
which only exists in build-systems-common.profile:

```console
$ git show --pretty='%h %ai %s' -s
b18120cc2 2022-08-09 21:11:18 +0000 new profile: gdu (#5289)
$ git grep -F -i projects -- etc
etc/profile-a-l/build-systems-common.profile:#whitelist ${HOME}/Projects
etc/profile-a-l/gitg.profile:#whitelist ${HOME}/YOUR_GIT_PROJECTS_DIRECTORY
etc/profile-m-z/nodejs-common.profile:# If you want whitelisting, change ${HOME}/Projects below to your node projects directory
etc/profile-m-z/nodejs-common.profile:#whitelist ${HOME}/Projects
$ git grep -F build-systems-common.profile -- etc
etc/profile-a-l/bundle.profile:include build-systems-common.profile
etc/profile-a-l/cargo.profile:include build-systems-common.profile
etc/profile-a-l/cmake.profile:include build-systems-common.profile
etc/profile-m-z/make.profile:include build-systems-common.profile
etc/profile-m-z/meson.profile:include build-systems-common.profile
etc/profile-m-z/pip.profile:include build-systems-common.profile
$ git grep -F nodejs-common.profile -- etc
etc/profile-m-z/node-gyp.profile:include nodejs-common.profile
etc/profile-m-z/node.profile:include nodejs-common.profile
etc/profile-m-z/npm.profile:include nodejs-common.profile
etc/profile-m-z/npx.profile:include nodejs-common.profile
etc/profile-m-z/semver.profile:include nodejs-common.profile
etc/profile-m-z/yarn.profile:include nodejs-common.profile
```

Or the comment about whitelisting paths other than the `${HOME}/Mail`
directory, which only exists in email-common.profile:

```console
$ git grep -F -C 1 'whitelist ${HOME}/Mail' -- etc
etc/profile-a-l/email-common.profile-# when storing mail outside the default ${HOME}/Mail path, 'whitelist' the custom path in your email-common.local
etc/profile-a-l/email-common.profile:whitelist ${HOME}/Mail
etc/profile-a-l/email-common.profile-whitelist ${RUNUSER}/gnupg
--
etc/profile-m-z/mutt.profile-whitelist ${HOME}/.w3m
etc/profile-m-z/mutt.profile:whitelist ${HOME}/Mail
etc/profile-m-z/mutt.profile-whitelist ${HOME}/mail
--
etc/profile-m-z/neomutt.profile-whitelist ${HOME}/.w3m
etc/profile-m-z/neomutt.profile:whitelist ${HOME}/Mail
etc/profile-m-z/neomutt.profile-whitelist ${HOME}/mail
$ git grep -F email-common.profile -- etc
etc/profile-a-l/claws-mail.profile:include email-common.profile
etc/profile-m-z/sylpheed.profile:include email-common.profile
```

> Also "makepkg" and "makedeb" are easy to confuse.

Do you mean because this comment in makepkg.profile mentions makepkg.local?

```firejail
# This profile could be significantly strengthened by adding the following to makepkg.local
# whitelist ${HOME}/<Your Build Folder>
# whitelist ${HOME}/.gnupg
```

I would assume that whatever applies to a given profile also applies to
whatever includes it.

And indeed, adding that line to makepkg.local would affect both profiles, since
makedeb.profile ultimately also includes makepkg.local.

> Also, unless you do weird things, makedeb and makepkg are mutually exclusive.
> They are meant to be installed on very different systems and things will
> diverge.

I don't see why either case would matter with regards to profile
maintainability.  As it stands on this PR, one profile already redirects to the
other anyway.

If you're takling about keeping `ignore include makepkg.local`, then I would be
curious as to when it could cause issues (considering that profiles are
generally written to be compatible with a wide variety of distros).

Note: I'm not 100% against keeping it, but I would like to at least understand
in which scenarios would it matter.

And if it is kept, it would be good to add a comment explaining why it's there
(since it's unusual to ignore a .local profile, as @rusty-snake mentioned).

> It's better to fork the comments.

A profile that looks like half of a fork of another profile while redirecting
to it at the same time seems confusing to me.

If the profiles diverge too much, why not fork the entire profile?

If they don't, why not write on the includer profile only what is actually
different about it (which usually just means `ignore` commands), like most
other redirect profiles?